docs/ovms_quickstart.md

```diff
@@ -134,3 +134,17 @@ or [PaddlePaddle model demo](../demos/classification_using_paddlepaddle_model/py
 
 Congratulations, you have completed the QuickStart guide. Try other Model Server [demos](../demos/README.md) or explore more [features](features.md) to create your application.
 Check also how to write the client code using [TFS API](./clients_tfs.md) and [KServe API](./clients_kfs.md)
+
```

Comment:
before 2025.2.1 publication, the WA for setting the timeout should be added to the docker command. I would drop the documentation of those variables in the quick start guide.